aip/0132.md

```diff
@@ -131,8 +132,13 @@ request message **should** contain a `string order_by` field.
   `"foo,bar"`.
 - The default sorting order is ascending. To specify descending order for a
   field, users append a `" desc"` suffix; for example: `"foo desc, bar"`.
-- Redundant space characters in the syntax are insignificant. `"foo, bar desc"`
-  and `" foo , bar desc "` are equivalent.
+- Redundant space characters in the syntax are insignificant.
+  `"foo, bar desc"`, `" foo , bar desc "`, and `"foo,bar desc"` are all
+  equivalent.
+- Subfields are specified with a `.` character, such as `foo.bar` or
```

Comment:
We don't address filtering on map<string, ...> fields here do we? If so we need to clarify the expectations for escape characters...

Reply:
We do not address this at this time. Perhaps we should, but that seems like a separate issue. (I do not actually know what the semantics are.)

Reply:
TODO added.